src/features/shields/index.ts

```diff
@@ -3,29 +3,117 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { SelectBox } from './select'
-import { Grid, GridLabel } from './grid'
-import { LinkIcon, ShieldIcon } from './icons'
+import { CloseButton, EmptyButton } from './button'
 import { Toggle } from './toggle'
-import { Stat, Link, Label, Highlight, UnHighlight, Description } from './text'
-import { ShieldsPanel, SiteCard, EnabledText, DisabledText, HeaderToggle, Header } from './wrappers'
+import { Option, Options } from './options'
+
+import {
```

Comment:
I believe you can save some repetition here by using the `export from ...` syntax for es6 / typescript